Contents/DefaultPrefs.json

```diff
@@ -31,7 +31,7 @@
 
     {
         "id": "SingleSeasonOrdering",
-        "label": "Use single season ordering (IMPORTANT: Make sure the setting in scanner is matches with this!)",
+        "label": "Use single season ordering (IMPORTANT: Make sure the setting in scanner matches with this!)",
```

Comment:
Generally, for a merge request, you wouldn't want to add unrelated stuff, but I'll let this through :)

Reply:
My brain didn't allow me to have a typo there. xD
Had to do it.